cron/config/config_test.go

```diff
@@ -29,6 +31,23 @@ const (
 	prodShardSize    int    = 250
 )
 
+func getByteValueFromFile(filename string) ([]byte, error) {
```

Comment:
Why not `ioutil.ReadFile()` ?

Reply:
Nice, thanks. Still kept this fn though to check for the case where filename is empty.